temporian/core/operators/since_last.py

```diff
@@ -82,9 +97,12 @@ def has_sampling(self) -> bool:
 def since_last(
     input: EventSetOrNode,
     sampling: Optional[EventSetOrNode] = None,
+    steps: int = 1,

```

Comment:
The default value (1) is defined in several places. What about only defining it in the Eventset::since_last ?

Reply:
Done.